GPU/Common/FramebufferCommon.cpp

```diff
@@ -40,16 +40,44 @@ void CenterRect(float *x, float *y, float *w, float *h, float origW, float origH
 		outW = frameW;
 		outH = frameH;
 	} else {
-		// Add special case for 1080p displays, cutting off the bottom and top 1-pixel rows from the original 480x272.
-		// This will be what 99.9% of users want.
-		if (origW == 480 && origH == 272 && frameW == 1920 && frameH == 1080 && !rotated) {
-			*x = 0;
-			*y = -4;
-			*w = 1920;
-			*h = 1088;
-			return;
+		bool fullScreenZoom = true;
+#ifdef _WIN32
```

Comment:
Not Windows specific.  Maybe MOBILE_DEVICE or something?

I see now this matches here, which is possibly also wrong?
https://github.com/hrydgard/ppsspp/blob/953f712e3a8efd615b1b40a1f25179fdf8a5d937/Core/Config.cpp#L438

SDL / Qt are not always full screen either (Mac, Linux.)

Also, I think we should rename `CenterRect` to like `CenterDisplayOutputRect` or something, to make sure it's never used for another purpose.

-[Unknown]